com.wso2telco.dep.gw.paymentapi/com.wso2telco.dep.gw.paymentapi.synapse/src/main/synapse-config/sequences/checkLimitSeq.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <sequence name="checkLimitSeq" onError="checkLimitError" trace="disable" xmlns="http://ws.apache.org/ns/synapse">
     <property expression="get-property('CONSUMER_KEY')" name="consumerKey" scope="default" type="STRING" xmlns:ns="http://org.apache.synapse/xsd"/>
-    <class name="com.wso2telco.dep.spend.limit.mediation.mediator.LimitCheckMediator"/>
+    <!--Commented following since current spendlimit handler has been deprecated-->
```

Comment:
It needs to be removed all spend limit related artifacts if existing spendlimit feature is deprecating. 
IMO, this sequence can be removed by doing small changes and it seems there are some other sequences related with spend limit feature.

Reply:
Commented this to preserve traceability and have minimum impact on change. Can remove spendlimit completely if needed.